hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -325,6 +329,8 @@ public void refreshTimeline() throws IOException {
       result = writeToSink(srcRecordsWithCkpt.getRight().getRight(),
           srcRecordsWithCkpt.getRight().getLeft(), metrics, overallTimerContext);
     }
+    String commitActionType = CommitUtils.getCommitActionType(cfg.operation, HoodieTableType.valueOf(cfg.tableType));
```

Comment:
shouldn't we log this at L462 ? 
https://github.com/apache/hudi/blob/773b3179834d6e0d2912a7a9d4b3b3873d0c2a21/hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java#L462

Reply:
Moved them to readFromSource.